templates/snyk-ecr-auto.template.yaml

```diff
@@ -64,6 +64,7 @@ Parameters:
   SnykAWSAccountNumber:
     Description: "Snyk's AWS Account ID which assumes a role in your account. Please see the deployment guide 'Technical requirements | Snyk Account Access' for this Account ID"
     Type: String
+    Default: "198361731867"
```

Comment:
These are left empty as they create cross account roles. To deploy the ID must be entered manually so that the person deploying acknowledges that this is a cross account role

Reply:
Ah, thanks, I didn't know that. I added them because the value is always the same, and ideally we want `taskcat test run` to work without any prerequisite steps. For that reason, I'll move this value over to `.taskcat.yml`.

Reply:
Done in [32a2162](https://github.com/aws-quickstart/quickstart-snyk-security/pull/31/commits/32a21625e523d3d42f048d671b383b3aa202903b).